src/core/apim_checkout_ec.tf

```diff
@@ -0,0 +1,68 @@
+##############
+## Products ##
+##############
+
+module "apim_checkout_ec_product" {
+  count  = var.checkout_enabled ? 1 : 0
```

Comment:
Is this count needed?

Reply:
ping :-)

Reply:
I left it in for consistency with current checkout configuration in [`apim_checkout.tf`](https://github.com/pagopa/pagopa-infra/blob/cb5eb907f66c808db4fde0f58e12d502b4263615/src/core/apim_checkout.tf)

Reply:
if we know that this APIs will be on all environments, better remove this count